### PR TITLE
Dont delete all docker images

### PR DESCRIPTION
**Pull request by lablans**
Only clean up docker images if bridgehead is actually running.

### CODE REVIEW

File: lib/functions.sh [hunk 1]
```diff
@@ -169,6 +169,17 @@ function retry {
   return 0
 }
 
+function bk_is_running {
+	detectCompose
+	RUNNING="$($COMPOSE -p bridgehead-$PROJECT -f ./$PROJECT/docker-compose.yml $OVERRIDE ps -q)"
+	NUMBEROFRUNNING=$(echo "$RUNNING" | wc -l)
+	if [ $NUMBEROFRUNNING -ge 2 ]; then
```
Comment (torbrenner) [inline lib/functions.sh:176]: why greater equals than 2? Shouldn't we also accept 1 running service?

File: lib/update-bridgehead.sh [hunk 2]
```diff
@@ -4,10 +4,15 @@ source lib/functions.sh
 AUTO_HOUSEKEEPING=${AUTO_HOUSEKEEPING:-true}
 
 if [ "$AUTO_HOUSEKEEPING" == "true" ]; then
-	A="Performing automatic maintenance: Cleaning docker images."
+	A="Performing automatic maintenance: "
+	if bk_is_running; then
+		A="$A Cleaning docker images."
+		docker system prune -a -f
```
Comment (PierreDelpy) [inline lib/update-bridgehead.sh:10]: sollen wir auch gleich ein volume prune ergänzen? (für ungenutzte volumes)

File: lib/functions.sh (hunk 1 above)
Comment (lablans) [reply lib/functions.sh:176]: That's the way `wc -l` works -- and there are no Bridgeheads with <2 services

File: lib/update-bridgehead.sh (hunk 2 above)
Comment (lablans) [reply lib/update-bridgehead.sh:10]: Not sure (requires testing). Please make another PR for that

**Pull request merged**